packages/frontend-2/.env.example

```diff
@@ -42,3 +42,5 @@ NUXT_PUBLIC_ENABLE_DIRECT_PREVIEWS=true
 
 # Ghost API
 NUXT_PUBLIC_GHOST_API_KEY=
+
+NUXT_WEBFLOW_API_TOKEN=your_webflow_api_token_here
```

Comment:
How is this supposed to work locally? Is everyone supposed to ask for a token? Or should we have a demo one for everyone to use?

Reply:
I've updated the env example to include the real key. This key (although stored in env) does not need to remain private, and has already been hard coded into the Connectors. It only has read access to Webflow.